app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -951,6 +953,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
+  @Config(qualifiers = "land-xhdpi")
```

Comment:
@rt4914 is landscape actually correct to use for these?

@veena14cs this might just be a case where the new UI isn't fitting within the screen on Robolectric & we now need more room in order for the test to pass. Does it pass with something like:

```kotlint
@Config(qualifiers = "w360dp-h760dp-port-xxxhdpi")
```

(I think that roughly corresponds to a Pixel XL)

Reply:
I did some changes in the UI and now tests passes without this.

Reply:
Marking as resolved since Rajat is taking a look.